backend/data/src/main/kotlin/io/tolgee/security/JwtToken.kt

```diff
@@ -0,0 +1,30 @@
+package io.tolgee.security
+
+import io.jsonwebtoken.Claims
+import io.jsonwebtoken.Jws
+import io.jsonwebtoken.Jwts
+import java.security.Key
+import java.util.*
+
+class JwtToken(private val value: String, private val key: Key?) {
+  private lateinit var _parsed: Jws<Claims>
+  private val parsed: Jws<Claims>
```

Comment:
Check this out.
https://kotlinlang.org/docs/delegated-properties.html#lazy-properties